.travis.yml

```diff
@@ -2,8 +2,46 @@ language: scala
 scala: 2.12.8
 
 before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
-install: jabba install adopt@1.8.192-12 && jabba use $_ && java -version
-script: sbt test
+install: 
+  - jabba install adopt@1.8.212-03 && jabba use $_ && java -version
+  - echo "Installing oc 3.11"
+  - curl -OL https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
+  - echo "4b0f07428ba854174c58d2e38287e5402964c9a9355f6c359d1242efd0990da3  openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz" | sha256sum -c
+  - tar --strip 1 -xvzf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz && chmod +x oc && sudo cp oc /usr/local/bin/ && rm oc
```

Comment:
why not `mv oc ...` instead of `cp`?

Reply:
No reason other than I copy/pasted most (if not all) of this from https://github.com/akka/akka-kubernetes-tests/blob/master/.travis.yml#L31 (but upgraded the version of `oc`.